### ISSUE TITLE
Can't get pandoc_reader working

### ISSUE THREAD

**Issue by mclearc** [liob/pandoc_reader#7]
I can't seem to get the plugin to work on pelican 3.6.3. I consistently see it choke when typogrify is set to 'true' and even with typogrify off it throws errors on all files. Specifically it says that it `could not find information about NameError: title`. Any help is appreciated. Thanks.

**Comment by liob** [comment]
Just tested with pelican 3.6.3. Here, everything works as expected. Did you set a title in the head of the markdown file? You can find a markdown header template here: http://docs.getpelican.com/en/3.6.2/content.html#file-metadata

**Comment by mclearc** [comment]
Thanks for the quick response. Turns out part of the problem lay with the YAML headers I was using. I didn't think to check whether Pandoc_reader could parse them (though I did have `+yaml_metadata_block` in the extensions settings. 

Non-YAML headers seem to work. My problem now is that I can't seem to get pandoc-citeproc to process citations. I've tried placing the `bibliography: file` in both the header and the `PANDOC_ARGS`. It doesn't seem to work in either. Any ideas?
